src/module.cpp

```diff
@@ -2296,6 +2296,9 @@ void Module::execPreprocessor(const char *infilename, llvm::raw_string_ostream *
     opts.addMacroDef("ISPC");
     opts.addMacroDef("PI=3.1415926535");
 
+    if (g->enableLLVMIntrinsics) {
+        opts.addMacroDef("ISPC_LLVM_INTRINSICS_DEFINED");
```

Comment:
I think `ISPC_LLVM_INTRINSICS_ENABLED` is a better name.

Reply:
Done!. I added some info in doc as well but the build script for doc fails with something above.